crypto/rand/drbg_lib.c

```diff
@@ -141,7 +153,7 @@ int RAND_DRBG_set(RAND_DRBG *drbg, int nid, unsigned int flags)
  */
 RAND_DRBG *RAND_DRBG_new(int type, unsigned int flags, RAND_DRBG *parent)
 {
-    RAND_DRBG *drbg = OPENSSL_zalloc(sizeof(*drbg));
+    RAND_DRBG *drbg = OPENSSL_secure_zalloc(sizeof(*drbg));
```

Comment:
oh ho!  This is new.  And probably worth mentioning in news or changes.
and does it work?  is secure heap set up by the time this function is called?

Reply:
Well, yes and no: For the three shared DRBGs it was already introduced in 933033b6928e9f01b99e7f2125f840bc052c5d96. This PR only moved it, see https://github.com/openssl/openssl/pull/5294/commits/e5b90a71246808fa13bce0bf59abc746e9bf2ad5#diff-9181ac017a6177a5f2619f65c9b7a346L768. So it's only new for e.g. for the SSL DRBGs. I hope there is no size limit on the secure heap or otherwise negative impact?